### DISCUSSION TITLE
Is there ability to set  Encoder.Quality when save image file

### DISCUSSION THREAD

**Discussion by burikus** [SixLabors/ImageSharp.Drawing#202] [Show and tell] [answered]
Hi!
At system.drawing there's an ability to set Encoder.Quality like
        
```
var encoderParameters = new EncoderParameters(1);
encoderParameters.Param[0] = new EncoderParameter(Encoder.Quality, 100L); //1L-100L
bitmap.Save(stream, GetEncoder(ImageFormat.Jpeg), encoderParameters);  //GetEncoder() returns ImageCodecInfo
```

Is there any mechanism in SixLabors for that ? Maybe you may share some link or idea where to look at?
(Yesterday the first time I heard about SixLabors - just need cross-platform support and don't want to use system.drawing.)

**Comment by JimBobSquarePants** [comment, marked as answer]
Why are you asking in the wrong repository (_that's 2X so far by the way, do it again and you will be blocked_) and why not simply read the documentation?

https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Formats.Jpeg.JpegEncoder.html#SixLabors_ImageSharp_Formats_Jpeg_JpegEncoder_Quality

https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Formats.Webp.WebpEncoder.html#SixLabors_ImageSharp_Formats_Webp_WebpEncoder_Quality

**Reply by burikus** [reply]
I've found these (I use it).  Thank you anyway.
